main.go

```diff
@@ -136,6 +138,12 @@ yab includes a full man page (man yab), which is also available online: http://y
 	setGroupDescs(parser, "transport", "Transport Options", toGroff(_transportOptsDesc))
 	setGroupDescs(parser, "benchmark", "Benchmark Options", toGroff(_benchmarkOptsDesc))
 
+	if err := plugin.AddToParser(pluginParserAdapter{parser}); err != nil {
+		for _, err := range multierr.Errors(err) {
+			out.Warnf("WARNING: Error adding plugin-based custom flags: %v. Continuing.", err)
+		}
```

Comment:
optional: Can do,

```
if err := plugin.AddToParser(..); err != nil {
  out.Warnf("WARNING: Error adding plugin-based custom flags: %+v", err)
}
```
multierr errors use a nice multi-line formatted message if `%+v` was used.

See the example in https://godoc.org/go.uber.org/multierr#Combine